### ISSUE TITLE
Autocomplete field options not available to assistive technology

### ISSUE THREAD

**Issue by bmbc-dev-detec** [LocalGovIMS/localgov-ims#334]
The autocomplete suggestions field isn't operable for keyboard users. When tested in VoiceOver and NVDA, the available options aren't read aloud and there's no indication to what kind of field this is.

When content can be operated through a keyboard or alternate keyboard, it is operable by people with no vision (who cannot use devices such as mice that require eye-hand coordination) as well as by people who must use alternate keyboards or input devices that act as keyboard emulators. Keyboard emulators include speech input software, sip-and-puff software, on-screen keyboards, scanning software and a variety of assistive technologies and alternate keyboards. Individuals with low vision also may have trouble tracking a pointer and find the use of software much easier (or only possible) if they can control it from the keyboard.

**Comment by bmbc-dev-detec** [comment]
Ready for retesting after completion of #416

**Comment by bmbc-dev-detec** [comment]
Closing. Will add any problem found after retesting as new issues
